examples/stable/mysql-csi-snapshot/README.md

```diff
@@ -0,0 +1,265 @@
+# Backup and restore MySQL DB using CSI VolumeSnapshot
+
+VolumeSnapshots provide Kubernetes users with a standardized way to copy a volume's contents at a particular point in time without creating an entirely new volume. This functionality enables, for example, database administrators to backup databases before performing edit or delete modifications.
+
+## Introduction
+This document explains how Kanister leverages the use of CSI VolumeSnapshots to capture and restore MySQL database.
```

Comment:
As discussed offline, let's use a non-DB application for this example. Generally, I assume that the blueprint would be used directly on PVCs but if we want to talk about an application, let's use timelogger.